test_conformance/CMakeLists.txt

```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
+
+#copy all csv files
```

Comment:
For style consistency with the rest of the file could you add a space and a capital here? e.g.
`# Copy all csv files`